### PR TITLE
Plo hotfixes

### PR DESCRIPTION
**Pull request by Delor4**

### CODE REVIEW

File: plostd.c [hunk 1]
```diff
@@ -34,34 +34,27 @@ int plostd_isalnum(char c)
...
 
-int plostd_strlen(char *s)
+int plostd_strlen(const char *s)
 {
-	char *p;
 	unsigned int k = 0;
 
-	for (p = s; *p; p++)
-		k++;
+	for (;*s; s++, k++);
```
Comment (kemonats) [inline plostd.c:41]: shorter
```
for (; s[k]; k++);
```

File: plostd.c [hunk 2]
```diff
@@ -34,34 +34,27 @@ int plostd_isalnum(char c)
...
 	return k;
 }
 
 
 int plostd_strcmp(const char *s1, const char *s2)
 {
-	unsigned int k = 0;
+	for(; *s1 && *s2 && *s1 == *s2; ++s1, ++s2);
```
Comment (kemonats) [inline plostd.c:48]: one condition less
```
for(; *s1 && *s1 == *s2; ++s1, ++s2);
```

**Comment by Delor4** [comment]
commit updated

File: plostd.c [hunk 3]
```diff
@@ -34,34 +34,27 @@ int plostd_isalnum(char c)
 }
 
 
-int plostd_strlen(char *s)
+int plostd_strlen(const char *s)
```
Comment (kemonats) [inline plostd.c:37]: better will be
```
unsigned int plostd_strlen(const char *s)
```
returned value `k` is type `unsigned int`

Comment (Delor4) [reply plostd.c]: sure

File: plostd.c [hunk 4]
```diff
@@ -34,34 +34,27 @@ int plostd_isalnum(char c)
...
 		return ERR_STD_NOTEQ;
-	return 0;
+	return ERR_NONE;
 }
 
-
 char *plostd_itoa(unsigned int i, char *buff)
 {
-	static char *digits = "0123456789";
+	const char *digits = "0123456789";
```
Comment (agkaminski) [inline plostd.c:57]: static const would be best

File: plostd.c [hunk 5]
```diff
@@ -100,7 +93,7 @@ char *plostd_itoa(unsigned int i, char *buff)
 
 char *plostd_itoah(u8 *ip, u8 is, char *buff, int lz)
 {
-	static char *digitsh = "0123456789abcdef";
+	const char *digitsh = "0123456789abcdef";
```
Comment (agkaminski) [inline plostd.c:96]: static const would be best

File: plostd.c [hunk 6]
```diff
@@ -122,7 +115,7 @@ char *plostd_itoah(u8 *ip, u8 is, char *buff, int lz)
 
 static unsigned int _plostd_ltoa(unsigned long n, unsigned int base, char *buff)
 {
-	static char *digits = "0123456789abcdef";
+	const char *digits = "0123456789abcdef";
```
Comment (agkaminski) [inline plostd.c:118]: static const would be best

File: plostd.c [hunk 7]
```diff
@@ -155,9 +148,9 @@ char *plostd_ltoa(unsigned long n, unsigned int base, char *buff)
 }
 
 
-unsigned int plostd_ahtoi(char *s)
+unsigned int plostd_ahtoi(const char *s)
 {
-	static char *digitsh = "0123456789abcdef";
+	const char *digitsh = "0123456789abcdef";
```
Comment (agkaminski) [inline plostd.c:153]: static const would be best

**Review by agkaminski — CHANGES_REQUESTED** [review]

File: plostd.c (hunk 4 above)
Comment (Delor4) [reply plostd.c]: changed to static const

**Pull request merged**